example.json

```
@@ -34,7 +34,7 @@
         }
     }
 
-    // Overide which completion data attribute is used to test when the 
+    // Overide which completion data attribute is used to test when the
```

```suggestion
    // Override which completion data attribute is used to test when the
```
Typical that the whitespace amend brings attention to the typo!